Sums.rb

```diff
@@ -0,0 +1,19 @@
+multipleThree = Array.new
+multipleFive = Array.new
+(0..999).each do |i|
+  if i % 3 == 0
+    multipleThree << i

```

Comment:
since you're just going to sum these up anyway, you might as well do the summation here. so you could do

```ruby
multipleThree = 0

(0..999).each do |i|
  if i % 3 == 0
    multipleThree += i